docs/index.md

```diff
@@ -0,0 +1,138 @@
+# uData
+
+[![Python Requirements Status][requires-io-badge]][requires-io-url]
+[![JavaScript Dependencies Status][david-dm-badge]][david-dm-url]
+[![JavaScript Development Dependencies Status][david-dm-dev-badge]][david-dm-dev-url]
+[![Join the chat at https://gitter.im/etalab/udata][gitter-badge]][gitter-url]
+
+
+_uData_ is an open and social data hub.
+
+## Compatibility
+
+udata requires Python 2.7, MongoDB 3.2+, ElasticSearch 1.7 and Redis.
+
+## Installation
+
+### System dependencies
+
+```shell
+sudo apt-get install -y --no-install-recommends tar git wget curl build-essential pkg-config  python python-dev python-pip libjpeg-dev zlib1g-dev libpng12-dev libtiff5-dev libfreetype6-dev liblcms2-dev libopenjpeg-dev libwebp-dev libpng12-dev libxml2-dev libxslt1-dev liblzma-dev libyaml-dev && sudo pip install virtualenv cython
+```
+
+!!! note
```

Comment:
Yes, it's using the [admonition plugin](https://pythonhosted.org/Markdown/extensions/admonition.html) which is working with the default mkdocs theme and the readthedocs one.